jest.config.json

```diff
@@ -21,7 +21,8 @@
   },
   "snapshotSerializers": ["enzyme-to-json/serializer"],
   "transform": {
-    "^.+\\.(js|jsx)?$": "<rootDir>/test/jestLib/babel-transformer.js"
+    "^.+\\.(js|jsx)?$": "<rootDir>/test/jestLib/babel-transformer.js",
+    "^.+\\.webapp$": "<rootDir>/json-transformer.js"
```

Comment:
Why no put this json-transformer in /test/jestLib/ ?

Reply:
:+1: